modin/core/storage_formats/pandas/utils.py

```diff
@@ -38,7 +39,12 @@ def get_default_chunksize(length, num_splits):
     )
 
 
-def compute_chunksize(df, num_splits, default_block_size=32, axis=None):
+def compute_chunksize(
+    df,
+    num_splits,
+    default_block_size=None,
```

Comment:
Since default value for this parameter was changed we should correct corresponding line for this parameter in the docstring.
`default_block_size : int, default: 32` -> `default_block_size : int, optional`